lib/shared/ColorPicker.js

```diff
@@ -15,8 +15,8 @@ class ColorPicker {
     this.applied = false;
 
     // bound by
-    this.updateCallback = () => {};
-    this.closeCallback = () => {};
+    this.updateCallback = () => {}
```

Comment:
I'm not too happy with these two; they're assignments of functions and usually these would be terminated with a `;`. This looks like an omission in `eslint`.